lib/index.js

```diff
@@ -128,18 +134,25 @@ util.inherits(SpawnMocha, EventEmitter);
 
 var mochaStream = function mocha(opts) {
   opts = opts || {};
+  var files = [];
```

Comment:
This `files` related logic is a stream you need to do in your own customs stream, it affects concurrency.  To optimize concurrency, you want the default to be splitting the jobs file by file.